lib/codeowners/cli/main.rb

```diff
@@ -20,6 +20,7 @@ class Main < Base
       # for pre-commit: --from HEAD --to index
       def check(repo = '.')
         @codeowners_changed = false
+        @ignore = { suggestions: false, added_files: false }
```

Comment:
As both values are false, any reason to store the false values?

I'd also do not intersect between the linters and just use a simple `@ignore = true` or even `@quit = true` and just set it on demand and consume it on demand.

Reply:
we can also build something later like `--only added` or `--except suggestions` to skip one of the steps if we want to skip step by step or execute a single action.

Reply:
I thought we could skip only one of the actions but let's simplify and build the options for executing just one of them 👍